### PR TITLE
README: update sponsors

### PR DESCRIPTION
**Pull request by selsta**
- http -> https
- remove outdated sponsors
- add new sponsors
- added a link to sponsorships page

### CODE REVIEW

**Review by dEBRUYNE-1 — APPROVED** [review]

**Review by dEBRUYNE-1 — APPROVED** [review]

**Review by dEBRUYNE-1 — APPROVED** [review]

**Review by erciccione — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -95,15 +95,13 @@ The Monero donation address is: `888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnj
...
-[<img width="100" src="https://static.getmonero.org/images/sponsors/dome9.png"/>](https://dome9.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/araxis.png"/>](https://araxis.com)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/jetbrains.png"/>](https://www.jetbrains.com/)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/navicat.png"/>](https://www.navicat.com/)
-[<img width="150" src="https://static.getmonero.org/images/sponsors/symas.png"/>](https://www.symas.com/)
+Core development funding and/or some supporting services are also graciously provided by [sponsors](https://www.getmonero.org/community/sponsorships/):
+
+[<img width="150" src="https://www.getmonero.org/img/sponsors/tarilabs.png"/>](https://tarilabs.com/)
+[<img width="150" src="https://www.getmonero.org/img/sponsors/globee.png"/>](https://globee.com/)
+[<img width="150" src="https://www.getmonero.org/img/sponsors/symas.png"/>](https://www.symas.com/)
```
Comment (jonathancross) [inline README.md:102]: Should be `https://symas.com/` to prevent unnecessary redirect.

File: README.md [hunk 2]
```diff
@@ -95,15 +95,13 @@ The Monero donation address is: `888tNkZrPN6JsEgekjMnABU4TBzc2Dt29EPAvkRxbANsAnj
...
+[<img width="150" src="https://www.getmonero.org/img/sponsors/forked_logo.png"/>](https://www.forked.net/)
```
Comment (jonathancross) [inline README.md:103]: Hmmm... this URL doesn't load correctly over `https` in Chrome.

Their TLS cert is issued to `portal.forked.net` so it fails in the browser.  If you go to that domain over https, it 302 redirects to the _insecure_ version.
image

**Review by jonathancross — CHANGES_REQUESTED** [review]

**Comment by selsta** [comment]
@jonathancross done

**Review by jonathancross — APPROVED** [review]
Thanks!

**Pull request merged**